### ISSUE TITLE
Redis Cluster Setup not Working

### ISSUE THREAD

**Issue by stephenjeba** [KarelCemus/play-redis#119]
Hi,
Redis cluster setup is not working.
version:1.5.1
I had configured as below
```
play.cache.redis 
   { 
    cluster: [{ 
      host:"<hostname>", 
      port:<port>
     }]
   }
```
By default it is taking localhost and 6379 as configuration. 

Do I making any mistake.

**Comment by KarelCemus** [comment]
Hi, is there any error or exception? Could be related to [missing password](https://github.com/KarelCemus/play-redis/pull/105#issuecomment-319195433)?

**Comment by KarelCemus** [comment]
Oh, actually, it looks like Redis Cluster is not supported by `1.5.x` :-(

**Comment by KarelCemus** [comment]
Released `1.6.0`
